spring/boot3-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/MappingWebEndpointPathMapper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring.actuate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.actuate.endpoint.EndpointId;
+import org.springframework.boot.actuate.endpoint.web.PathMapper;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.util.StringUtils;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+/**
+ * Fork of <a href="https://github.com/be-hase/spring-boot/blob/24925c3daef169deaa9b2107ef02be4a67edb886/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/MappingWebEndpointPathMapper.java">MappingWebEndpointPathMapper</a>.
```

Comment:
nit: We also need to add the license from the file.
[This](https://github.com/line/armeria/blob/920eca8abeeffe3ab9293e866a6d27acdc6ee468/core/src/main/java/com/linecorp/armeria/common/ClientCookieDecoder.java#L17) is an example.

Reply:
thanks, fixed.
https://github.com/line/armeria/pull/4880/commits/d4ed172c220f5fa3bfea0f420a85e10d1e8cf33b